acapy_wallet_upgrade/tests/e2e/cases.py

```diff
@@ -141,7 +144,6 @@ async def credentials_v1(
             requested_attributes=requested_attributes,
         )
         assert alice_pres_ex_askar.state == "verified"
-        assert alice_pres_ex_askar.verified == "true"
```

Comment:
I assume these assertions were removed because redundant in the context of the tests

Reply:
This was just because the minimum controller library supplied by indicio was updated and this property didn't exist anymore.